resources/services/sql/managed_databases.go

```diff
@@ -0,0 +1,373 @@
+package sql
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/Azure/azure-sdk-for-go/services/preview/sql/mgmt/v4.0/sql"
+	"github.com/cloudquery/cq-provider-azure/client"
+	"github.com/cloudquery/cq-provider-sdk/provider/schema"
+)
+
+func SqlManagedDatabases() *schema.Table {
+	return &schema.Table{
+		Name:         "azure_sql_managed_databases",
+		Description:  "ManagedDatabase a managed database resource.",
+		Resolver:     fetchSqlManagedDatabases,
+		Multiplex:    client.SubscriptionMultiplex,
+		DeleteFilter: client.DeleteSubscriptionFilter,
```

Comment:
As this is a relation on instances there is no need for Multiplex nor DeleteFilter.